ofp/packet.go

```diff
@@ -109,7 +112,7 @@ func (p *PacketIn) ReadFrom(r io.Reader) (int64, error) {
 	// Read the packet-in header, then the list of match
 	// rules, that used to match the processing packet.
 	return encoding.ReadFrom(r, &p.Buffer, &p.Length,
-		&p.Reason, &p.Table, &p.Cookie, &p.Match, &defaultPad2)
+		&p.Reason, &p.Table, &p.Cookie, &p.Match, &defaultPad2, &p.Data)
```

Comment:
I believe this data should be _written_ as well (in method ```WriteTo```).

Reply:
Yes, of course. Realized that while fixing the tests :-)